deploy/yamls/zfs-driver.yaml

```diff
@@ -724,6 +723,7 @@ spec:
       containers:
         - name: csi-node-driver-registrar
           image: quay.io/k8scsi/csi-node-driver-registrar:v1.2.0
+          imagePullPolicy: IfNotPresent
```

Comment:
We should probably remove this line.. as default will be `IfNotPresent` ?

Reply:
that is correct. other places specifically mention "IfNotPresent" for imagePullPolicy. So made it in sync with other places.